akka-docs-dev/rst/java/stream-testkit.rst

```diff
@@ -0,0 +1,25 @@
+.. _stream-testkit-java:
+
+###############
+Testing streams
+###############
+
+Akka Streams comes with an :mod:`akka-stream-testkit` module that provides tools which can be used for controlling and asserting various parts of the stream pipeline.
+
+Probe Sink
+==========
+
```

Comment:
`Sink.actorRef` works in many cases and then using ordinary actor testkit should be familiar to many users